code/modules/tutorials/flock_tutorial.dm

```diff
@@ -0,0 +1,421 @@
+		..()
+
+	SetUp()
+		..()
+		var/turf/T = get_turf(src.ftutorial.center)
+		for (var/obj/flock_structure/cage/cage in range(3, T))
+			qdel(cage.occupant)
+			qdel(cage)
+		var/obj/flock_structure/ghost/tealprint = new(T, src.ftutorial.fowner.flock, /obj/flock_structure/relay)
+		tealprint.fake = TRUE
+
+/datum/tutorialStep/flock/showcase
+	name = "Structure showcase"
+	instructions = "Here are all the flock structures you can create, along with a shadow of the relay, your ultimate goal. Click the exit tutorial button in the bottom right corner to exit the tutorial."
+	SetUp()
+		..()
+		src.ftutorial.fowner.abilityHolder.addAbility(/datum/targetable/flockmindAbility/tutorial_exit)
+		var/datum/mapPrefab/allocated/prefab = get_singleton(/datum/mapPrefab/allocated/flock_showcase)
+		var/datum/allocated_region/region = prefab.load()
+		for (var/turf/T in REGION_TILES(region))
+			var/obj/spawner/flock_structure/structure_spawner = locate() in T
+			structure_spawner?.spawn_structure(ftutorial.fowner.flock)
+			var/mob/living/carbon/human/bad_immortal/fake_tdummy = locate() in T
+			if (fake_tdummy)
+				src.ftutorial.fowner.flock.updateEnemy(fake_tdummy)
+		for(var/turf/T in landmarks[LANDMARK_TUTORIAL_START])
+			if(region.turf_in_region(T))
+				src.ftutorial.fowner.set_loc(T)
+				break
+		src.ftutorial.ui_interact(src.ftutorial.fowner)
+
+/mob/living/intangible/flock/flockmind/verb/help_my_tutorial_is_being_a_massive_shit()
+	set name = "EMERGENCY TUTORIAL STOP"
+	if (!src.tutorial)
+		boutput(src, "<span class='alert'>You're not in a tutorial, doofus. It's real. IT'S ALL REAL.</span>")
+		return
+	src.tutorial.Finish()
+	src.tutorial = null
+
+//for debug, do not enable on live or it will cause runtimes and break everything
+// /mob/living/intangible/flock/flockmind/verb/skip_tutorial_step()
+// 	set name = "SKIP TUTORIAL STEP"
+// 	src.tutorial.Advance()
+
+/obj/machinery/junk_spawner
+	var/stuff = list(/obj/item/extinguisher, /obj/item/crowbar, /obj/item/wrench)
+	process(mult)
+		if (!(locate(/obj/item) in get_turf(src)))
+			var/type = pick(src.stuff)
+			new type(src.loc)
+
+/mob/living/carbon/human/normal/chef/shoot_gun_person
+	New()
+		..()
+		var/obj/item/gun/kinetic/zipgun/gun = new(src)
+		gun.failure_chance = 0
+		src.put_in_hand(gun)
+		SPAWN(0)
+			while(TRUE)
+				for (var/dir in modulo_angle_to_dir)
+					if (is_incapacitated(src))
+						return
+					src.set_dir(dir)
+					gun.ammo.amount_left = 2
+					var/turf/target = get_step(src, dir)
+					gun.shoot(target, src.loc, src)
+					sleep(1.5 SECONDS)
+
+/mob/living/carbon/human/bad_immortal
+	real_name = "Target dummy"
+	Life(datum/controller/process/mobs/parent)
+		. = ..()
+		for (var/obj/item/implant/I in implant) //no infinite item stacks
+			if (istype(I, /obj/item/implant/projectile))
+				I.on_remove(src)
+				implant.Remove(I)
+				qdel(I)
```

Comment:
Are there tables or? Not sure how these would be cut out

Reply:
`full_heal` drops all negative implants on the floor, building up an infinite stack of items

Reply:
Is that a bug with full_heal then or?

Reply:
Darts from the gnesis turret in this case.

Reply:
Sounds like a bug? Darts should be implants

Reply:
Yes, and `full_heal` ejects them along with shrapnel and such.